crate/memories/src/sqlite_mem.rs

```diff
@@ -31,6 +31,7 @@ impl From<async_sqlite::Error> for SqliteMemoryError {
     }
 }
 
+pub const FINDEX_TABLE_NAME: &str = "findex_memory";
```

Comment:
Since changing the repo structure is a breaking change, you can remove this constant and parameterize each memory instantiation function by a name.
```suggestion
```